frontend/src/components/widget/Lookup/Lookup.js

```diff
@@ -300,7 +302,7 @@ class Lookup extends Component {
       rowId,
       tabIndex,
       validStatus,
-      recent,
+      // recent,
```

Comment:
Because it was only used to pass an empty array. Don't ask me why: https://github.com/metasfresh/metasfresh/pull/10056/files#diff-92a02f4e1c97e8c29d0c3efc69b4917fL695